src/main/kotlin/org/anefdev/mixbuddy/service/MixBuddyService.kt

```diff
@@ -0,0 +1,243 @@
+package org.anefdev.mixbuddy.service
+
+import org.anefdev.mixbuddy.model.MusicPlaylist
+import org.anefdev.mixbuddy.model.MusicTrack
+import org.anefdev.mixbuddy.model.SpotifyUser
+import lombok.NoArgsConstructor
+import lombok.SneakyThrows
+import org.anefdev.mixbuddy.conf.SpotifyConfig
+import org.anefdev.mixbuddy.util.PlaylistParser
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import se.michaelthelin.spotify.SpotifyApi
+import se.michaelthelin.spotify.SpotifyHttpManager
+import se.michaelthelin.spotify.enums.AuthorizationScope
+import se.michaelthelin.spotify.model_objects.specification.Paging
+import se.michaelthelin.spotify.model_objects.specification.PlaylistTrack
+import se.michaelthelin.spotify.model_objects.specification.Track
+import java.net.URI
+import java.util.*
+
+@Service
+@NoArgsConstructor
+class MixBuddyService(
+    private val config: SpotifyConfig,
+    private val playlistParser: PlaylistParser
+) {
+    private val LOGGER: Logger = LoggerFactory.getLogger(MixBuddyService::class.java)
+
+    private var allPlaylists: List<MusicPlaylist>? = null
```

Comment:
im not sure if this is going to work in the intended way, spring beans are singleton by default, what is going to happen when more than one user is trying to interact with the service?